spec/factories/account_invitations.rb

```diff
@@ -2,8 +2,8 @@
   factory :account_invitation do
     association :user, factory: :user
     name { FFaker::Name.first_name }
-    email { FFaker::Internet.email }
-    token { FFaker::Lorem.phrase }
+    email { user.email }
```

Comment:
Is it an invitation to themselves? Why changed it?

```suggestion
    email { FFaker::Internet.email }
```